spec/factories/application_choice.rb

```diff
@@ -43,7 +43,7 @@
 
       after(:build) do |application_choice, _evaluator|
         application_choice.status = :interviewing
-        application_choice.interviews << build(:interview, provider: application_choice.current_provider)
+        application_choice.interviews << build(:interview, provider: application_choice.current_course_option.provider)
```

Comment:
🤔  wonder why current provider didn't work?

Reply:
I think it's because it's an after(_:build_), and there's just too much indirection.